TLTagsContol/TLTagsControl.m

```diff
@@ -78,10 +78,11 @@ - (void)commonInit {
     tagInputField_.layer.borderColor = [UIColor lightGrayColor].CGColor;
     tagInputField_.backgroundColor = [UIColor whiteColor];
     tagInputField_.delegate = self;
-    tagInputField_.font = [UIFont fontWithName:@"HelveticaNeue" size:14];
     tagInputField_.placeholder = @"tag";
     tagInputField_.autocorrectionType = UITextAutocorrectionTypeNo;
     
+    _font = [UIFont fontWithName:@"HelveticaNeue" size:14];
```

Comment:
I guess we should set this font to the tagInputField_ as well.

Could you please update your pull request?

`tagInputField_.font = _font;` would be fine